applications/projects/SofaPhysicsAPI/SofaPhysicsSimulation.cpp

```diff
@@ -55,7 +56,13 @@ SofaPhysicsAPI::SofaPhysicsAPI(bool useGUI, int GUIFramerate)
 
 SofaPhysicsAPI::~SofaPhysicsAPI()
 {
-    delete impl;
+    std::cout << "SofaPhysicsAPI::~SofaPhysicsAPI()" << std::endl;
```

Comment:
This looks like debugging so it should probably be removed or replaced with msg_info(...)